src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -305,6 +280,31 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  "/stream/transactions":
+    post:
+      operationId: StreamTransactions
+      summary: Get Transactions Stream
+      description: |
+        Returns transactions which have been committed to the ledger.
+        The returned response is in a paginated format, ordered by most recently committed.
```

Comment:
that isn't always true. There's a parameter which can control ordering.

Reply:
Good catch!